tools/check_volume_levels.m

```diff
@@ -183,18 +188,41 @@ function plot_levels(meas, tc, lm)
 	hold off;
 	xlabel('Time (s)');
 	ylabel('Gain (dB)');
+	grid on;
 end
 
 function pass = check_levels(meas, tc, lm, verbose)
 	pass = 1;
+	dg_tol = 0.1;
 	gains =  meas.levels - lm.sine_dbfs;
 	sv = size(tc.volumes);
 	for j = 1:sv(2)
 		for i = 1:sv(1)
+			% Initial location to test
```

Comment:
could you please add a space between `ts = tc.vctimes(i)+tc.meas(1); `as `ts = tc.vctimes(i) + tc.meas(1);`